lib/modules/manager/circleci/extract.ts

```diff
@@ -63,7 +63,8 @@ export function extractPackageFile(content: string): PackageFile | null {
         dep.versioning = 'docker';
         if (
           !dep.depName?.startsWith('ubuntu-') &&
-          !dep.depName?.startsWith('windows-server-')
+          !dep.depName?.startsWith('windows-server-') &&
+          !dep.depName?.startsWith('android')
```

Comment:
```suggestion
          !dep.depName?.startsWith('android:')
```

Reply:
it doesn't work with colon, that was my first intuition too.

Reply:
Should it be a strict equals?

Reply:
im not that familiar with circleci, but from what i gathered from their docs it might be.
https://circleci.com/docs/2.0/android-machine-image/#no-orb-example

Reply:
so check both equal for `android` only and `android-` prefix for te other?

Reply:
i searched github for [`android-` ](https://github.com/search?p=4&q=path%3A%2F.circleci%2F+filename%3Aconfig.yml+android-&type=Code) and it doesnt seems to be a thing.
while `android` without the dash yields much more results.
WDYT?

Reply:
I think it's ok if we check for exactly android or starting with `android-`, in case they add that scheme in future.